src/main/java/org/cryptomator/ui/error/ErrorController.java

```diff
@@ -100,8 +131,75 @@ public void copyDetails() {
 		});
 	}
 
-	/* Getter/Setter */
+	private void loadHttpResponse(HttpResponse<InputStream> response){
+		if (response.statusCode() == 200) {
+			Map<String,ErrorDiscussion> map = new Gson().fromJson(
```

Comment:
Rename the variable to what it contains, e.g. `errorDiscussions` or `errorDiscussionsMap`.

Reply:
renamed to errorDiscussionsMap